test/body.html

```diff
@@ -58,8 +58,7 @@
 ]
 }});</script><script>AF_initDataCallback({key: 'ds:30', isError:  false , hash: '27', data:function(){return [["cin:acc","https://myphonenumbers-pa.clients6.google.com"]
 ]
-}});</script><script>AF_initDataCallback({key: 'ds:31', isError:  false , hash: '28', data:function(){return [["cic:sc",["AF","DZ","AO","BH","BD","BJ","BF","KH","CM","CG","CD","CI","GH","GU","GN","ID","IQ","IL","JO","KZ","KE","KW","KG","LA","LR","MG","MW","MY","MV","MA","MZ","NE","NG","OM","PK","PS","PH","SA","SN","SC","SL","SO","ZA","LK","TZ","TH","TG","TN","TR","UG","UA","UZ","VN","ZM"]
-]
+}});</script><script>AF_initDataCallback({key: 'ds:31', isError:  false , hash: '37', data:function(){return [["cic:vd","104889609452098951178","joao.marques.antunes@gmail.com",0,""]
```

Comment:
What is the significance of `104889609452098951178` and `joao.marques.antunes@gmail.com` here?

Reply:
I edited the patch and left the original text on the line and am able to connect.

Reply:
@dmidthun that is because this wont effect the running of the program, just the tests.

@smammy I assumed it was so that test would pass unline #102 which fails https://travis-ci.org/yakyak/hangupsjs/builds/445768280

Reply:
OK, got it. Thanks.

Reply:
The email is from a fake account that I used to generate the tests manually.

The tests need to be greatly improved as they do close to nothing.